### PR TITLE
Feature 47 use spdx licenses

### PR DESCRIPTION
**Pull request by mbjones**
This PR is to modify the Dataset.md guide to add a new section `License` in which we recommend people use SPDX license URIs to unambiguously specify the license for data and metadata use, as described in issue #47. The PR also includes an Architecture Decision Record describing the proposal.

### CODE REVIEW

File: decisions/47-use-spdx-licenses.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  },
+  "@id": "http://www.sample-data-repository.org/dataset/123",
+  "@type": "Dataset",
+  "name": "Removal of organic carbon by natural bacterioplankton communities as a function of pCO2 from laboratory experiments between 2012 and 2016",
+  <strong>"license": "http://spdx.org/licenses/CC0-1.0"</strong>
+  ...
+}
+</pre>
+
+While many licenses are ambiguous about the license URI for the license, the Creative Commons licenses are an exception in that they provide extremely consistent URIs for each license, and these are in widespread use.  So, while we recommend using the SPDX URI, we recognize that some sites may want to use the CC license URIs directly, which is acceptable.  Here's an example using the traditional CC URI for the license.
```
Comment (datadavev) [inline decisions/47-use-spdx-licenses.md]: ... they provide ~~extremely~~ consistent URIs ...

~~So, while we recommend using the SPDX URI, we recognize that some sites may want to use the CC license URIs directly, which is acceptable.~~ **While we recommend using the SPDX URI, it is acceptable to use the CC license URIs directly if preferred.**

File: decisions/47-use-spdx-licenses.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+&lt;div class="optional-license-text"&gt;
+   &lt;p&gt;Apache License
+  &lt;br /&gt;
+
+Version 2.0, January 2004
+  &lt;br /&gt;
+...
+</pre>
+
+Finally, the SPDX project provides structured data files of the [SPDX license data](https://github.com/spdx/license-list-data) in machine readable formats, including turtle and json-ld.  These could be imported into COR or other vocabulary servers if we want a queryable graph of the license data.
```
Comment (datadavev) [inline decisions/47-use-spdx-licenses.md]: ... vocabulary servers ~~if we want~~ **to provide** a queryable graph of the license data.

File: decisions/47-use-spdx-licenses.md (hunk 1 above)
Comment (mbjones) [reply decisions/47-use-spdx-licenses.md]: Resolved in sha af17b3d.

File: decisions/47-use-spdx-licenses.md (hunk 2 above)
Comment (mbjones) [reply decisions/47-use-spdx-licenses.md]: Resolved in sha af17b3d.

File: decisions/47-use-spdx-licenses.md (hunk 1 above)
Comment (ashepherd) [reply decisions/47-use-spdx-licenses.md]: should the license be a resource?:
```
"license": { "@id": "http://spdx.org/licenses/CC0-1.0" }
```

**Comment by ashepherd** [comment]
actually, on second thought maybe it's easier for publishers to just specify the URL, and let the harvester determine if its a resource or a literal

**Comment by mbjones** [comment]
@ashepherd I added your suggested changes and edits in sha 98c67d4 along with other clarifications based on the discussion in issue #47.  I think this PR is ready to merge into `develop` unless there are further comments or changes desired.

**Pull request merged**